src/main/java/org/myrobotlab/service/InMoovHead.java

```diff
@@ -97,12 +97,14 @@ public boolean connect(String port, Integer headYPin, Integer headXPin, Integer
         error("controller for head is not connected");
       }
     }
-    ((Servo) neck).attach(controller, headYPin, neck.getRest(), neck.getVelocity());
-    ((Servo) rothead).attach(controller, headXPin, rothead.getRest(), rothead.getVelocity());
-    ((Servo) jaw).attach(controller, jawPin, jaw.getRest(), jaw.getVelocity());
-    ((Servo) eyeX).attach(controller, eyeXPin, eyeX.getRest(), eyeX.getVelocity());
-    ((Servo) eyeY).attach(controller, eyeYPin, eyeY.getRest(), eyeY.getVelocity());
-    ((Servo) rollNeck).attach(controller, rollNeckPin, rollNeck.getRest(), rollNeck.getVelocity());
+    

```

Comment:
I guess the assumption is that if you want to use a DiyServo here, it would just ignore the attach method?

Reply:
that's all been cleaned up...
(Servo) cast has been there forever, and would explode if a different ServoControl was used.